src/components/Nav/Nav.spec.js

```diff
@@ -1,14 +1,15 @@
 import React from 'react';
 import Enzyme, { mount } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
+import { BrowserRouter as Router } from 'react-router-dom';
 import Nav from './Nav';
 
 Enzyme.configure({ adapter: new Adapter() });
 
 describe('Nav Mounts', () => {
   it('renders', () => {
     const wrapper = mount(
-      <Nav />,
+      <Router><Nav /></Router>,
```

Comment:
Same question with home, this doesn't make sense to me--does it fix a broken test?